src/index.html

```diff
@@ -20,9 +19,9 @@ <h1 class="list-title">Today's To Do</h1>
       <form action="#" id="list-addition" class="flex">
         <label for="item"></label>
         <input type="text" id="item" name="item" placeholder="Add your new item..." class="new-list">
-        <span class="material-symbols-outlined font-color">
+        <span class="material-symbols-outlined font-color" id="pressed">
```

Comment:
- [ optional] dont you think it will be better without the `subdirectory text`

Reply:
The subdirectory text is the icon